config/locales/es.yml

```diff
@@ -300,7 +298,9 @@ es:
       header: "dependencias de %{title}"
     gem_members:
       authors_header: Autores
-      owners_header: Propietarios
+      owners_header: Administradores
```

Comment:
Sorry, didn't notice that push and yank by translation were already there. Undo `owners_header` change only. push and yank by is fine.